tests/lib/snaps.sh

```diff
@@ -43,3 +43,23 @@ install_generic_consumer() {
     snap install --dangerous generic-consumer/*.snap
     rm -rf generic-consumer
 }
+
+is_classic_confinement_supported() {
```

Comment:
We already ask snapd if strict confinement is supported or not. Asking for classic confinement would be just another option.